woocommerce/woocommerce-android
# Load a single list in orders tab and apply any filter selection

Work in progress: #4934 

### Description

 - Removed code for the 2 tabs from order list sections
 - Load a single list and apply any selected filters to that list ALWAYS

Known issues in this changes that have been addressed here: #5239
- Any bug related to search functionality
- Toolbar shadow appearing after opening filters and then going back to order list
- Navigate back from filter flow should display a dialog to discard changes only if filters have changed

### Testing instructions
- Open order list
- Play adding and removing filters to the list 
- Check displayed results are correct
- Kill the app and reopen and check filters are still applied

### Images/gif

https://user-images.githubusercontent.com/2663464/141773547-559c21b8-fcf8-479d-891e-f2c94c2f8fd9.mp4
